apps/src/templates/studioHomepages/Sections.jsx

```diff
@@ -1,91 +0,0 @@
-import React from 'react';
```

Comment:
This file was split into the `TeacherSections` and `StudentSections` components.